tests/test_dsp.py

```diff
@@ -212,6 +213,66 @@ def test_regu_inversion(impulse):
     npt.assert_allclose(res.freq[:, -1], [0.25])
 
 
+def test_time_shift():
```

Comment:
I'm leaving this as is, makes spotting issues easier

Reply:
I think it would help to split it in different test for the purpose of spotting issues (or use `@pytest.mark.parametrize()`, which does that automatically)

Reply:
👍